component/libfuse/libfuse_handler.go

```diff
@@ -364,7 +364,7 @@ func (lf *Libfuse) fillStat(attr *internal.ObjAttr, stbuf *C.stat_t) {
 func libfuse_getattr(path *C.char, stbuf *C.stat_t, fi *C.fuse_file_info_t) C.int {
 	name := trimFusePath(path)
 	name = common.NormalizeObjectName(name)
-	// log.Trace("Libfuse::libfuse_getattr : %s", name)
+	log.Trace("Libfuse::libfuse_getattr : %s", name)
```

Comment:
comment this back, getattr is called for a very large number of times in the system and doing any logging here will hurt the performance.

Reply:
We need to comment this back.